scripts-available/CDB_TransformToWebmercator.sql

```diff
@@ -36,7 +36,7 @@ BEGIN
 
   -- Don't bother clipping if the geometry boundary doesn't
   -- go outside the valid extent.
-  IF latlon_input @ valid_extent THEN
+  IF @postgisschema@.geometry_within(latlon_input, valid_extent) THEN
```

Comment:
?

Reply:
CDB_TransformToWebmercator uses the postgis operator `@` which is under the schema public. Since I didn't know how to change `@` to something like `public.operator@(geometry,geometry)` I changed it to the underlying Postgis function so I could add the `@postgisschema@ in front. Without the proper schema restoring some triggers might fail since  the pg_dump options change the search_path (removing public).